Makefile

```diff
@@ -43,7 +43,6 @@ licenses: ## Verifies dependency licenses and requires GITHUB_TOKEN to be set
 	golicense hack/license-config.hcl karpenter
 
 apply: ## Deploy the controller into your ~/.kube/config cluster
-	kubectl create ns karpenter || true
 	helm template karpenter charts/karpenter --namespace karpenter \
```

Comment:
don't we need `--create-namespace` for a fresh install to work?

Reply:
This workflow assumes that the namespace isn't created or deleted. I think it's much safer to just assume that the developer manages this namespace since there are so many ways it could be created (helm, eksctl, etc) that have bitten us. Worst case, a new dev needs to run `kubectl create namespace karpenter`